dbt/adapters/trino/connections.py

```diff
@@ -183,8 +186,14 @@ def http_scheme(self):
     def method(self):
         return "kerberos"
 
-    def trino_auth():
-        return trino.auth.KerberosAuthentication()
+    def trino_auth(self):
+        os.environ["KRB5_CLIENT_KTNAME"] = self.password
+        return trino.auth.KerberosAuthentication(
+            config = self.config,
+            service_name = self.type,
+            principal = self.principal,
+            ca_bundle= self.cert
```

Comment:
What's the reasoning of passing `self.type` to `service_name`?

Reply:
That make sense but it can be configured in `http-server.authentication.krb5.service-name` so the value can be different from `trino`.  The better idea would be to set the default value to `trino` and make this field configurable as an optional parameter.